docs/dinamicko-programiranje/knapsack.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Knapsack
+---
+
+### Problem
+
+Ruksak kapaciteta $x$ moramo napuniti sa stvarima kojih ima $n$, a od kojih svaka zauzima određeni kapacitet i ima određenu vrijednost. Naravno cilj nam je maksimizirati ukupnu vrijednost stvari u ruksaku. U zadatku bitnu razliku čini jesu li stvari tekuće ili čvrste.
+
+### Rješenje
+
+Ako su stvari tekuće onda pohlepno možemo uzimati stvari što veće vrijednosti ili što većeg omjera količine i vrijednosti ovisno o tome kako je zadan zadatak.
```

Comment:
Kakav je zadatak u kojem se uzima samo vrijednost u ozbir?
Omjer je obrnut, dakle omjer vrijednosti i količine.

Reply:
Imas pravo, vrijednost ima smisla samo ako je jednaka kolicina svih stvari, popravio sam sad to